edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/Constants.cs

```diff
@@ -18,6 +18,7 @@ public class Constants
 
         public const string IotEdgeIdentityCapability = "iotEdge";
         public const string ServiceIdentityRefreshMethodName = "RefreshDeviceScopeIdentityCache";
+        public const string IoTEdgeProductInfoIdentifier = "IoTEdge";
```

Comment:
Should this be `EdgeHub`?

Reply:
I was thinking about it.. wasn't sure if we should keep it generic (this came from an Edge device) or make it specific (from EdgeHub on the Edge device).

Reply:
Changed to EdgeHub